tests/lvmo/test_lvm_node_reboot_catalogsource.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import tier4, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM, wait_for_lvm_pod_running
+from ocs_ci.ocs.exceptions import CommandFailed, CatalogSourceNotFoundAfterReboot
+from ocs_ci.utility.vsphere_nodes import VSPHERENode
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.deployment.deployment import create_catalog_source
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
+@skipif_lvm_not_installed
+@skipif_ocs_version("<4.11")
+class TestLvmCatalogSourceNodeReboot(ManageTest):
+    """
+    Test lvm snapshot bigger than disk
+
+    """
+
+    number_of_reboots = 10
+
+    def test_node_reboot_sno_catalogsource(
+        self,
+    ):
+        """
+        test reboot multiple time to check catalogsource existence
+        .* Reboot node
+        .* Check that catalogsource exists
+        .* repeat number_of_reboots
+
+        """
+        lvm = LVM()
+
+        image = lvm.image
+        catalogsource_not_found_occurrences = 0
+        for reboot in range(1, (self.number_of_reboots + 1)):
+            node = VSPHERENode(host=lvm.ip, user="core")
```

Comment:
VSPHERENode is not supposed to be used directly from the test. For such operations, we should use ocs_ci.ocs.platform_nodes.VMWareNodes.

A reference for such usage could be found in tests/manage/z_cluster/nodes/test_nodes_restart.py